influxdb.tf

```diff
@@ -52,3 +52,20 @@ data "template_file" "influxdb_values" {
     influxdb_admin_pass  = "${var.influxdb_admin_pass}"
   }
 }
+
+resource "influxdb_database" "efd" {
+  name = "efd"
+
+  retention_policies = [
+    {
+      name     = "year"
+      duration = "52w"
+      default  = "true"
+    },
+  ]
+
+  depends_on = [
+    "helm_release.nginx_ingress",
```

Comment:
I don't think this dep is necessary as this resource shouldn't have an interaction with ingress.

Reply:
I agree `helm_release.influxdb` already has a dependency on `helm_release.nginx_ingress` so this one can be removed.